### PR TITLE
notebook: GLM rolling regression (run with pymc v5) 

### PR DESCRIPTION
**Pull request by reshamas**
# Description
Run notebook with pymc v5

+ [ ] Notebook follows style guide https://docs.pymc.io/en/latest/contributing/jupyter_style.html
+ [ ] PR description contains a link to the relevant issue:
  * a tracker one for existing notebooks (tracker issues have the "tracker id" label)
  * or a proposal one for new notebooks
+ [ ] Check the notebook is not excluded from any pre-commit check: https://github.com/pymc-devs/pymc-examples/blob/main/.pre-commit-config.yaml

### Helpful links
* https://github.com/pymc-devs/pymc-examples/blob/main/CONTRIBUTING.md

### CODE REVIEW

**Comment by reshamas** [comment]
@OriolAbril 
Not sure what the issue is in doc building failure, as I reran the notebook and didn't make any significant changes.

**Comment by OriolAbril** [comment]
I think sphinx-codeautolink had a new release, can you try adding a <latest release pin in `requirements-docs.txt`?

**Comment by OriolAbril** [comment]
There already is a patch release of codeautolink and the build works again

File: examples/generalized_linear_models/GLM-rolling-regression.myst.md [hunk 1]
```diff
@@ -250,6 +250,7 @@ cb.ax.set_yticklabels(ticklabels);
 
 - Created by [Thomas Wiecki](https://github.com/twiecki/)
 - Updated by [Benjamin T. Vincent](https://github.com/drbenvincent) June 2022
+- Run PyMC v5 by Reshama Shaikh, January 2023
 
 +++
```
Comment (OriolAbril) [inline examples/generalized_linear_models/GLM-rolling-regression.myst.md:256]: Can be merged already, but ideally I would remove aeppl from the watermark

**Review by OriolAbril — APPROVED** [review]

**Comment by reshamas** [comment]
Ah, Oriol, I was rerunning the notebook to remove the aeppl watermark.

**Pull request merged**